src/jest/index.ts

```diff
@@ -179,6 +179,16 @@ function configureTsConfig(options: JestOptions): Rule {
       module: 'commonjs',
       emitDecoratorMetadata: true,
       allowJs: true,
+    }, {
+      // add the following type entries
+      types: ['jest']
+        .concat(tsConfigContent.compilerOptions?.types ?? [])
+        .filter(
+          (type: string) =>
+            // remove the following type entries
+            ['jasmine'].indexOf(type) === -1
```

Comment:
Wouldn't `type !== "jasmine"` be simpler?

Reply:
That would be. In this case I opted to use `indexOf` to allow for easy expansion of the excluded types list.